source/2021-11-06-the-ember-times-issue-167.md

```diff
@@ -1,27 +1,26 @@
 ---
 title: The Ember Times - Issue No. 167
-author: the crowd
+author: Chris Ng, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2020
-alias : "blog/2021/11/06-the-ember-times-issue-167.html"
+alias : "blog/2020/11/06-the-ember-times-issue-167.html"
```

Comment:
This `alias` is correct, but the filename for this file is wrong. The year is `2021` but it should be `2020`.

Reply:
good catch fixed that!